### PR TITLE
Screenshots endpoint

### LINKED ISSUE #14: Screenshot - takeScreenshot endpoint
Endpoint details: https://api.helioviewer.org/docs/v2/api/api_groups/screenshots.html#takescreenshot

### LINKED ISSUE #15: Screenshot - downloadScreenshot endpoint
Endpoint details: https://api.helioviewer.org/docs/v2/api/api_groups/screenshots.html#downloadscreenshot

### PR DESCRIPTION
**Pull request by akash5100**
* closes #14 
* closes #15

### CODE REVIEW

File: hvpy/api_groups/screenshots/take_screenshot.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+    Attributes
+    ----------
+    {Shared}
+    date : datetime.datetime
+        Desired date/time of the image.
+
+    imageScale : float
+        Image scale in arcseconds per pixel.
+
```
Comment (nabobalis) [inline hvpy/api_groups/screenshots/take_screenshot.py:22]: I think these line breaks should be removed.

File: hvpy/api_groups/screenshots/take_screenshot.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    callback: Optional[str] = None
+    _date_vaidator = validator("date", allow_reuse=True)(convert_date_to_isoformat)
+
+    def get_output_type(self) -> OutputType:
+        """
+        Returns the output type of the API call.
+        """
+        if self.display:
+            return OutputType.RAW
+        return OutputType.JSON
```
Comment (nabobalis) [inline hvpy/api_groups/screenshots/take_screenshot.py:111]:
```suggestion
        return OutputType.RAW if self.display else OutputType.JSON
```

Maybe?

File: hvpy/api_groups/screenshots/take_screenshot.py (hunk 1 above)
Comment (akash5100) [reply hvpy/api_groups/screenshots/take_screenshot.py:22]: Do you mean the changes in the docstring you did #45?

File: hvpy/api_groups/screenshots/take_screenshot.py (hunk 2 above)
Comment (akash5100) [reply hvpy/api_groups/screenshots/take_screenshot.py:111]: Done.

**Review by nabobalis — APPROVED** [review]

**Pull request merged**